paparazzi-junit4/build.gradle

```diff
@@ -0,0 +1,129 @@
+import com.android.build.gradle.internal.publishing.AndroidArtifacts
+
+apply plugin: 'org.jetbrains.kotlin.jvm'
+apply plugin: 'org.jetbrains.kotlin.plugin.compose'
+apply plugin: 'org.jetbrains.dokka'
+apply plugin: 'com.vanniktech.maven.publish'
+apply plugin: 'aar2jar'
+
+configurations {
+  layoutlibRuntime {
+    canBeConsumed = false
+    attributes.attribute(AndroidArtifacts.ARTIFACT_TYPE, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+  layoutlibResources {
+    canBeConsumed = false
+    attributes.attribute(AndroidArtifacts.ARTIFACT_TYPE, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+}
+
+aar2jar.configureForConfiguration("compileOnly")
+aar2jar.configureForConfiguration("testImplementation")
+
+dependencies {
+  registerTransform(org.gradle.api.internal.artifacts.transform.UnzipTransform) {
+    from.attribute(AndroidArtifacts.ARTIFACT_TYPE, ArtifactTypeDefinition.JAR_TYPE)
+    to.attribute(AndroidArtifacts.ARTIFACT_TYPE, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+}
+
+dependencies {
+  testImplementationAarAsJar libs.androidx.compose.ui.android
+  compileOnlyAarAsJar libs.androidx.compose.ui.android
+  compileOnlyAarAsJar libs.androidx.compose.runtime
```

Comment:
calling out this switcheroo; replacing `compileOnlyAarAsJar libs.androidx.activity` with this: https://github.com/cashapp/paparazzi/pull/1462/files#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87df

Reply:
nice, much better dependency